packaging/os/redhat_subscription.py

```diff
@@ -65,12 +65,34 @@
             - Specify a subscription pool name to consume.  Regular expressions accepted.
         required: False
         default: '^$'
+    consumer_type:
+        description:
+            - The type of unit to register, defaults to system
+        required: False
+        default: null
+        version_added: "2.x"  # TBD
+    consumer_name:
+        description:
+            - Name of the system to register, defaults to the hostname
+        required: False
+        default: null
+        version_added: "2.x"  # TBD
+    consumer_id:
+        description:
+            - The existing system data is pulled from the server
```

Comment:
Should have be more descriptive of what the consumer id is in addition to the effect of specifying it.